doc/LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Scala is licensed under the [BSD 3-Clause License](http://opensource.org/licenses/BSD-3-Clause).
+Scala is licensed under the [Apache License Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
 
 ## Scala License
```

Comment:
Should this be dropped, or reworded?  It's not longer the "Scala License", based on BSD 3-Clause, it's now the Apache Licence, Version 2.0, as is.